charts/testkube-enterprise/README.md

```diff
@@ -272,7 +272,7 @@ A Helm chart for Testkube Enterprise
 | testkube-cloud-api.api.email.fromName | string | `""` | Name to use for sending outgoing emails |
 | testkube-cloud-api.api.features.disablePersonalOrgs | bool | `false` | Toggle whether to disable personal organizations when a user signs up for the first time |
 | testkube-cloud-api.api.features.legacyTests | bool | `false` | Toggle whether to enable support for legacy tests (Test, TestSuite) |
-| testkube-cloud-api.api.inviteMode | string | `"auto-accept"` | Configure which invitation mode to use (email|auto-accept): email uses SMTP protocol to send email invites and auto-accept immediately adds them |
+| testkube-cloud-api.api.inviteMode | string | `"email"` | Configure which invitation mode to use (for now email is the only supported option): email uses SMTP protocol to send email invites |
```

Comment:
If you do not want to use e-mail you can still use `"none"`, which is useful if you do not have a SMTP server configured.